apps/condo/domains/meter/schema/RegisterMetersReadingsService.test.js

```diff
@@ -1102,7 +1061,7 @@ describe('RegisterMetersReadingsService', () => {
         ]
         const [data] = await registerMetersReadingsByTestClient(adminClient, o10n, readings)
 
-        const metersReadings = await MeterReading.getAll(adminClient, { meter: { id_in: data.map((row) => row.meter.id) } })
+        const metersReadings = await MeterReading.getAll(adminClient, { meter: { id_in: data.map((row) => row.meter.id) } }, { sortBy: 'createdAt_ASC' })
         expect(metersReadings).toEqual([
```

Comment:
Maybe it's better to use `expect.arrayContaining([...])` instead of sorted array?